haystack/document_stores/sql.py

```diff
@@ -8,6 +8,8 @@
 
 import numpy as np
 
+from generalimport import is_imported
+
 try:
```

Comment:
We can remove this `try/catch` now since we're using `generalimport` right?

Reply:
Yes but I would like to remove all instances of these `try-catch` in another PR. There are tons of them everywhere

Reply:
Ok, good idea.